lib/node_modules/@stdlib/blas/ext/base/gcusumkbn/benchmark/benchmark.js

```diff
@@ -21,14 +21,22 @@
 // MODULES //
 
 var bench = require( '@stdlib/bench' );
-var randu = require( '@stdlib/random/base/randu' );
+var uniform = require( '@stdlib/random/array/uniform' );
 var isnan = require( '@stdlib/math/base/assert/is-nan' );
 var pow = require( '@stdlib/math/base/special/pow' );
 var gfill = require( '@stdlib/blas/ext/base/gfill' );
+var Float64Array = require( '@stdlib/array/float64' );
 var pkg = require( './../package.json' ).name;
 var gcusumkbn = require( './../lib/main.js' );
 
 
+// VARIABLES //
+
+var options = {
+	'dtype': 'float64'
```

Comment:
This is fine. Alternatively, you'd use `generic` and then instead of `Float64Array`, you'd use, e.g., `array/base/zeros`.